src/mustache.erl

```diff
@@ -261,18 +261,20 @@ remove_space_from_tail_impl(_, Size) ->
     Size.
 
 %% @doc Number to binary
--spec to_binary(number() | binary() | string()) -> binary() | string().
+-spec to_binary(number() | binary() | string() | atom()) -> binary() | string().
 to_binary(Integer) when is_integer(Integer) ->
     list_to_binary(integer_to_list(Integer));
 to_binary(Float) when is_float(Float) ->
     io_lib:format("~p", [Float]);
+to_binary(Atom) when is_atom(Atom) ->
+    list_to_binary(atom_to_list(Atom));
 to_binary(X) ->
-    X.
+    iolist_to_binary(X).
```

Comment:
The float result would still be a string, not a binary.

But to be sure, does that matter for line 277? I made it `iolist_to_binary(X)` because `to_binary` is used in `escape` to create a binary. And actually, now that I look at it maybe float needs to be converted to a binary too?